Cannot find module '../package.json'

```
~/puppet-xp   test ±  git push                 
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../package.json'
Require stack:
- ~/puppet-xp/node_modules/@chatie/git-scripts/dist/bin/pre-push.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (~/puppet-xp/node_modules/@chatie/git-scripts/dist/bin/pre-push.js:48:24)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/binsee/CodeWork/puppet-xp/node_modules/@chatie/git-scripts/dist/bin/pre-push.js'
  ]
}
Failed to exec pre-push hook script
error: failed to push some refs to 'github.com:binsee/puppet-xp.git'
```

--- a/bin/pre-push.ts
+++ b/bin/pre-push.ts
@@ -10,6 +10,7 @@
  *
  */
 import shell from 'shelljs'
+import path from 'path'
 
 const NO_HOOK_VAR = 'NO_HOOK'
 const INNER_PRE_HOOK = 'CHATIE_INNER_PRE_HOOK'
@@ -55,7 +56,8 @@ if (refs?.[0]?.localCommit.match(/^0+$/)) {
   process.exit(0)
 }
 
-const packageVersion = require('../package.json').version
+const pkgFile = path.join(process.cwd(), 'package.json')
+const packageVersion = require(pkgFile).version
 const lastCommitMsg = shell.exec('git log --pretty=format:"%s" HEAD^0 -1', { silent : true }).stdout
 
 if (packageVersion === lastCommitMsg) {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@chatie/git-scripts",
-  "version": "0.7.1",
+  "version": "0.7.2",
   "description": "Git Hooks Integration for Chatie Projects",
   "directories": {
     "doc": "docs",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@chatie/git-scripts",
-  "version": "0.7.2",
+  "version": "0.7.3",
   "description": "Git Hooks Integration for Chatie Projects",
   "directories": {
     "doc": "docs",

